liquid-error/src/error.rs

```diff
@@ -94,6 +94,15 @@ pub struct Error {
     inner: Box<InnerError>,
 }
 
+impl<R> From<pest::error::Error<R>> for Error
```

Comment:
So you'll find we have a lack of `From`s.  This is because they become part of the public API and I'm trying to avoid implementation details in the public API.